src/components/IconGrid.jsx

```diff
@@ -3,23 +3,28 @@ import styled from 'styled-components'
 import IconTile from '../components/IconTile'
 
 const StyledIconGrid = styled.div`
-  max-width: 68.75rem;
-  margin: 0 auto;
-  display: grid;
   width: 100vw;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
-  justify-items: center;
-  gap: 1.5rem;
+  max-width: 1432px;
```

Comment:
let's turn this into `rem` as previously discussed. 

Formula is pixel you want / base pixel (we almost _always_ use 16 pixels) and that will equal the `rem` you want. 

So 1432 / 16 = 89.5 rem. 

I use `cmd + space` all the time and do the calculations directly in spotlight search. 

I also have a [VS Code plugin](https://marketplace.visualstudio.com/items?itemName=sainoba.px-to-rem) that converts the pixels automatically if I hit a command.